### PR TITLE
example [lambda_safe_deployments] use nodejs8.10 for the PreTraffic hook

### PR DESCRIPTION
**Pull request by sullis**
*Issue #, if available:*

n/a

*Description of changes:*

PreTrafficHook example now uses NodeJS 8.10 runtime

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by sullis** [comment]
awesome

File: examples/2016-10-31/lambda_safe_deployments/src/preTrafficHook.js [hunk 1]
```diff
@@ -1,47 +1,44 @@
...
-        }
-	});
+    try {
+      await codedeploy.putLifecycleEventHookExecutionStatus(params).promise();
+      console.log("putLifecycleEventHookExecutionStatus done. executionStatus=[" + params.status + "]");
+      return 'Validation test succeeded'
+    }
+    catch (err) {
+      console.log("putLifecycleEventHookExecutionStatus ERROR: " + err);
+      return 'Validation test failed'
```
Comment (brettstack) [inline examples/2016-10-31/lambda_safe_deployments/src/preTrafficHook.js]: This should be `throw new Error('Validation test failed')`

**Comment by sullis** [comment]
WDYT?  @brettstack

**Comment by jlhood** [comment]
@sullis Thanks for the update! I changed the base branch of this PR to develop instead of master. Can you rebase your changes onto latest develop so it can be merged?

**Comment by sullis** [comment]
> @sullis Thanks for the update! I changed the base branch of this PR to develop instead of master. Can you rebase your changes onto latest develop so it can be merged?


Done.

**Pull request merged**